CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- **2020-01-29**
```

Comment:
Just a reminder that when we decide to merge, this could be updated to the current day.

```suggestion
- **2020-02-XX**
```

Reply:
Good idea. I don't understand GitHub well enough; does that require my intervention?

Reply:
Yes, since I don't have direct write access to chi-lambda:interactive-mode branch, you'll need to update it (the "suggestion" here was not really needed and can be ignored). Or, I can update it separately after the merge. Actually I meant to highlight the date in README.md too but forgot. It's not that important anyway, just a tiny detail to avoid confusion.